1.14.0 (#266)
MetaMask/github-tools: CHANGELOG.md

## [Unreleased]

## [1.14.0]

### Added

> - accept release ota branch ([#263](https://github.com/MetaMask/github-tools/pull/263))

Reviewer: ```suggestion
- Support OTA branches in `release-branch-sync` action ([#263](https://github.com/MetaMask/github-tools/pull/263))
```

> - feat(actions): add unified Playwright test health reporting for extension and mobile ([#262](https://github.com/MetaMask/github-tools/pull/262))

Reviewer: ```suggestion
- Add `playwright-test-health-report` action for creating reports for Playwright tests, and posting to Slack ([#262](https://github.com/MetaMask/github-tools/pull/262))
```

### Fixed

> - every sync into a given release branch reuses the same branch ([#264](https://github.com/MetaMask/github-tools/pull/264))

Reviewer: Something like this?
```suggestion
- Allow `release-branch-sync` action to reuse branch ([#264](https://github.com/MetaMask/github-tools/pull/264))
```
